lighthouse-core/gather/drivers/cri.js

```diff
@@ -44,7 +44,7 @@ class CriDriver extends Driver {
         chromeRemoteInterface({port: port, chooseTab: tab}, chrome => {
           this._tab = tab;
           this._chrome = chrome;
-          this.beginLogging();
+          this._eventEmitter = chrome;
```

Comment:
can you add a comment to clarify that `chrome` already inherits eventemitter?

Also maybe we should rename that variable now? `cri` maybe?